src/professions/admin/qualifications.controller.ts

```diff
@@ -170,7 +170,7 @@ export class QualificationsController {
       ukRecognitionUrl: qualification?.ukRecognitionUrl,
       otherCountriesRecognition: qualification?.otherCountriesRecognition,
       otherCountriesRecognitionUrl: qualification?.otherCountriesRecognitionUrl,
-      isUK: isUK(version.occupationLocations),
+      isUK: isUK(version.occupationLocations || []),
```

Comment:
Would this be better?:

```suggestion
      isUK: version.occupationLocations ? isUK(version.occupationLocations) : false,
```

Reply:
Ah, yep, this is better!